### ISSUE TITLE
Fusionauth Freemarker templates with React

### ISSUE THREAD

**Issue by saul-gush** [FusionAuth/fusionauth-issues#1168]
### Description
For about 4 months now, we have been trying to integrate React SSR into the Fusionauth API. We have tried all sorts of methods including using Razzle and CRA. This is important for us to keep a consistent UI style across our platform using our UI components. The problem with Freemarker templates, it is crazy hard to do... a few of us engineers have been trial and error trying...and we are still battling with it. It would be great to have an example of or suggestions how this should work. 

The one method that we found, we need to upload the js files somewhere - I think the Fusionauth Theme API (https://fusionauth.io/docs/v1/tech/apis/themes/#create-a-theme)  only allows the theme Freemarker template. I should imagine we need to host the rest of the static files elsewhere? We tried to bundle the whole lot into one file but reach file size limits. 

So in short, this is really hard to do... any guidance/new tools here would be appreciated.

**Comment by mooreds** [comment]
Hi @saul-gush 

Thanks for using FusionAuth! 

You have a couple of options here. The quickest way to go would be for you to buy a support contract. You can [learn more here](https://fusionauth.io/pricing/editions/). This will get you direct access to the engineering team. Not trying to do the hard sell, but if you've been working on this off and on for 4 months, buying a support contract will likely be money well spent.

You also can post in the [community forums](https://fusionauth.io/community/forum/) with your question as well, and see if anyone has any advice. You can learn more about the difference between the forums (community support) and a support contract in the [tech support FAQ](https://fusionauth.io/technical-support/).

Finally, this is a support topic, which is better suited for either of the above options, not a bug report or feature request. So I'm going to close out this issue.
